pandas/tests/io/json/test_pandas.py

```diff
@@ -1154,38 +1154,32 @@ def test_sparse(self):
         assert expected == ss.to_json()
 
     @pytest.mark.parametrize(
-        "ts",
+        "ts,expected",
         [
-            Timestamp("2013-01-10 05:00:00Z"),
-            Timestamp("2013-01-10 00:00:00", tz="US/Eastern"),
-            Timestamp("2013-01-10 00:00:00-0500"),
+            (Timestamp("2013-01-10 05:00:00Z"), '"2013-01-10T05:00:00.000+00:00"'),
+            (
+                Timestamp("2013-01-10 00:00:00", tz="US/Eastern"),
+                '"2013-01-10T00:00:00.000-05:00"',
+            ),
+            (Timestamp("2013-01-10 00:00:00-0500"), '"2013-01-10T00:00:00.000-05:00"'),
         ],
     )
-    def test_tz_is_utc(self, ts):
+    def test_tz_utc_offsets(self, ts, expected):
         from pandas.io.json import dumps
 
-        exp = '"2013-01-10T05:00:00.000Z"'
-
-        assert dumps(ts, iso_dates=True) == exp
+        assert dumps(ts, iso_dates=True) == expected
         dt = ts.to_pydatetime()
-        assert dumps(dt, iso_dates=True) == exp
+        assert dumps(dt, iso_dates=True) == expected
 
-    @pytest.mark.parametrize(
-        "tz_range",
-        [
-            pd.date_range("2013-01-01 05:00:00Z", periods=2),
-            pd.date_range("2013-01-01 00:00:00", periods=2, tz="US/Eastern"),
```

Comment:
These tests were unfortunately asserting the previous wrong behavior, so I split them up into tests looking for UTC versus those looking for offsets